apps/builder/app/builder/features/style-panel/shared/property-name.tsx

```diff
@@ -215,6 +206,45 @@ const TooltipContent = ({
   );
 };
 
+export const PropertyTooltip = ({
+  openWithClick = false,
+  title,
+  description,
+  properties,
+  style,
+  onReset,
+  children,
+}: {
+  openWithClick?: boolean;
+  title?: string;
+  description?: React.ReactNode;
+  properties: StyleProperty[];
+  style: StyleInfo;
+  onReset?: undefined | (() => void);
+  children: ReactElement;
+}) => {
+  const [isOpen, setIsOpen] = useState(false);
+  return (
+    <Tooltip
+      open={isOpen}
+      onOpenChange={setIsOpen}
+      onClick={() => setIsOpen(false)}
```

Comment:
do I understand correctly that you want to close the tooltip when user clicks on it?

Reply:
Yes, but all click handlers on content will work

Reply:
I am not sure that clicking on a tooltip should close the tooltip at all ... did @taylornowotny describe this somewhere?

Reply:
Is there a reason why closing the tooltip on click improves the user experience?

A "see more" button has been tentatively planned for long CSS content, but we can go with some other pattern if there's a reason to.

Reply:
Other than the "see more" button I can't think of a reason to close or not close tooltips on click

Reply:
Radix closes tooltips by default on click because they are not interactive. Our tooltips have interactive elements, so if user misclicks a link or button tooltip would close.

It feels like when tooltip is interactive it shouldn't close when you click somewhere in the content that is not a button ...